packages/analytics-js/src/services/ErrorHandler/ErrorHandler.ts

```diff
@@ -102,7 +122,11 @@ class ErrorHandler implements IErrorHandler {
     let normalizedError = error;
     // Enhance error message
     if (isTypeOfError(error)) {
```

Comment:
How about this?

```
normalizedError = new Error('');
if (isTypeOfError(error)) {
  Object.assign(normalizedError, error);
}
normalizedError.message = errorMessage;
```

Reply:
We shouldn't do this. This will modify the original stack trace.

Reply:
It wouldn't as we're assigning all the properties from the original error object.

Reply:
I checked and it is modifying the stack.

Reply:
`Object.assign` is not adequate for Error type.

A quick Google search resulted in `Object.create`.

You can try this:
```
normalizedError = new Error(errorMessage);
if (isTypeOfError(error)) {
  normalizedError = Object.create(error, {
    message: { value: error.message }
  });
}
```

Reply:
What is the problem with the current implementation?

Reply:
We are not able to update the error object with the desired error message.

Reply:
We don't have to, the error stack is more important than the message and this will also update the stack so I am not implementing it.

Reply:
It is fine for such a few errors.

Reply:
Is it still not updating the stack? I tried the last solution, and it worked.

Can you try once again, and let's connect if you are unable to implement it?

Reply:
Addressed